README.md

```diff
@@ -1,2 +1,30 @@
 # hw01
 Repository for homework 01
+
+Hong Jin Jo
+====================
+
+Auto-biography
+---------------------
+
+This is a brief auto-biography of Hong Jin Jo.
+
+<img src="https://github.com/hongjinjo/myrepo/blob/master/JO_HONGJIN.jpg?raw=true">
+
+**Hong Jin Jo** was born in Seoul, Korea in 1988. He has attended the following schools:
+
+### Educational Background
+
+* Dongja Elementary School
+* Jayang Middle School
+* Korean Minjok Leadership Academy
+* Seoul National University
+
+He has acquired two sociology master degree from two institutions.
+
+1. Seoul National University
+2. The University of Chicago
+
+### Current Status
+
+Now, he is currently in his Sociology Ph.D. course at the University of Chicago. You can find more information about him at [here](https://sociology.uchicago.edu/directory/hong-jin-jo).
```

Comment:
Topic | Excellent | Satisfactory | Needs Work
-- | -- | -- | --
Coding style | X |   |  
Coding strategy | X |   |  
Presentation | X |   |  
Achievement, creativity |   | X |  
Ease of access | X |   |  